src/iosource/af_packet/src/Plugin.h

```diff
@@ -0,0 +1,17 @@
+// See the file "COPYING" in the main distribution directory for copyright.
+
+#pragma once
```

Comment:
Do we need this file at all? I see in the storage framework you have some `Plugin.cc` only directories. I could see this here, too, but might as well deserve a broader discussion.

Reply:
iosource/pcap only has Plugin.cc also. I'll drop this out.